lib/src/boilerplate_suggestors/advanced_props_and_state_class_migrator.dart

```diff
@@ -42,26 +41,30 @@ class AdvancedPropsAndStateClassMigrator extends GeneralizingAstVisitor
 
     final extendsFromCustomClass = !extendsFromUiPropsOrUiState(node);
     final hasMixins = node.withClause != null;
-
-    // Don't operate if the props class uses mixins and extends a custom class,
-    // unless the flag has been set.
-    if (hasMixins &&
-        extendsFromCustomClass &&
-        !shouldMigrateCustomClassAndMixins) return;
+    final convertedParentClass =
+        node.extendsClause.superclass.name.name + 'Mixin';
 
     final className = stripPrivateGeneratedPrefix(node.name.name);
     final newDeclarationBuffer = StringBuffer()
+      ..write('\n\n')
+      // Write a fix me comment if this class extends a custom class
+      ..write(!extendsFromCustomClass
+          ? ''
+          : '''
+          // FIXME: Verify that:
+          //   1. $convertedParentClass does not implement additional mixins.
+          //   2. any other mixins found here do not implement additional mixins.
```

Comment:
These instructions are pretty vague on needs to be done, and so is the term "implement".

I would  expect the first bullet to read as
> Ensure that all mixins used by ADifferentPropsClass are also mixed into this class.
ADifferentPropsClassMixin shouldn't implement other mixins, right? All of those should have shifted to ADifferentPropsClass?

> any other mixins found here do not implement additional mixins
What case is this talking about? Wouldn't the analyzer warn you about this? If so, we should just say "Fix any analyzer warnings on this class about missing mixins"